docs/tutorial.md

```diff
@@ -0,0 +1,602 @@
+```
+
+To retrieve information such as the username, password, and database. Enter:
+```shell
+juju run-action data-integrator/leader get-credentials --wait
+```
+This should output something like:
+```yaml
+unit-data-integrator-1:
+  UnitId: data-integrator/1
+  id: "28"
+  results:
+    mysql:
+      endpoints: 10.234.188.135:3306
+      password: FbflReIypeRhDH4UZ90pbUvi
+      read-only-endpoints: 10.234.188.214:3306
+      username: relation-4
+      version: 8.0.32-0ubuntu0.22.04.2
+    ok: "True"
+  status: completed
+  timing:
+    completed: 2023-01-29 23:11:17 +0000 UTC
+    enqueued: 2023-01-29 23:11:15 +0000 UTC
+    started: 2023-01-29 23:11:16 +0000 UTC
+```
+*(Note: your hostnames, usernames, and passwords will likely be different.)*
+
+### Access the related database
+Use `endpoints`, `username`, `password` from above to connect newly created database `test-database` on MySQL server:
+```shell
+> mysql -h 10.234.188.135 -P 3306 -urelation-4 -pFbflReIypeRhDH4UZ90pbUvi -e "show databases"
++--------------------+
+| Database           |
++--------------------+
+| test-database      |
++--------------------+
+```
+
+The newly created database `test-database` is also available on all other MySQL cluster members:
+```shell
+> mysql -h 10.234.188.214 -P 3306 -urelation-4 -pFbflReIypeRhDH4UZ90pbUvi -e "show databases"
++--------------------+
+| Database           |
++--------------------+
+| test-database      |
++--------------------+
+```
+
+When you relate two applications Charmed MySQL automatically sets up a new user and database for you.
+Please note the database name we specified when we first deployed the `data-integrator` charm: `--config database-name=test-database`.
+
+### Remove the user
+To remove the user, remove the relation. Removing the relation automatically removes the user that was created when the relation was created. Enter the following to remove the relation:
+```shell
+juju remove-relation mysql data-integrator
+```
+
+Now try again to connect to the same MySQL you just used in [Access the related database](#access-the-related-database):
+```shell
+mysql -h 10.234.188.135 -P 3306 -urelation-4 -pFbflReIypeRhDH4UZ90pbUvi -e "show databases"
+```
+
+This will output an error message:
+```
+ERROR 1045 (28000): Access denied for user 'relation-4'@'_gateway.lxd' (using password: YES)
+```
+As this user no longer exists. This is expected as `juju remove-relation mysql data-integrator` also removes the user.
+Note: data stay remain on the server at this stage!
+
+Relate the the two applications again if you wanted to recreate the user:
+```shell
+juju relate data-integrator mysql
+```
+Re-relating generates a new user and password:
+```shell
+juju run-action data-integrator/leader get-credentials --wait
+```
+You can connect to the database with this new credentials.
+From here you will see all of your data is still present in the database.
+
+## Transcript Layer Security (TLS)
```

Comment:
```suggestion
## Transport Layer Security (TLS)
```

Reply:
@MiaAltieri , we need to fix this in [MongoDB](https://github.com/canonical/mongodb-operator/blob/main/documentation/tutorial.md#transcript-layer-security-tls) operator too... 

Fixed for MySQL.